src/SimpleJavaApp/MainApp.java

```diff
@@ -1,32 +1,70 @@
 package SimpleJavaApp;
 
 import javax.swing.*;
+import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
 
 
-
-public class MainApp {
+public class MainApp extends JFrame {
     private JButton Login;
     private JPanel panelMain;
-    private JPasswordField passwordField1;
-    private JTextField textField1;
+    private JPasswordField password;
```

Comment:
You should explicitly name the variable to distinguish view and value `passwordView` for view and `password` for value